### PR TITLE
Removing unused database columns related to XeThru configuration (CU-wf8f4e).

### PR DESCRIPTION
**Pull request by mariocimet**

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-wf8f4e remove led, noisemap, sensitivity columns from locations table](https://app.clickup.com/t/wf8f4e)

File: db/db.js [hunk 1]
```diff
@@ -375,33 +374,31 @@ async function isOverdoseSuspected(xethru, session, location, clientParam) {
...
+    if (condition1 == 1) {
       alertReason = ALERT_REASON.STILLNESS
     }
 
     const conditionThreshold = 1
 
     // This method just looks for a majority of conditions to be met
     // This method can apply different weights for different criteria
-    if (condition1 + condition2 + condition3 >= conditionThreshold) {
+    if (condition1 + condition2 >= conditionThreshold) {
```
Comment (trisapeace) [inline db/db.js:401]: We never ended up using this weighting. 
What do you think about getting rid of this concept and using more readable names for the conditions and ditching `conditionThreshold`?

File: index.js [hunk 2]
```diff
@@ -637,9 +637,6 @@ app.post(
       'fallbackPhones',
       'heartbeatPhones',
       'twilioPhone',
-      'sensitivity',
-      'led',
-      'noiseMap',
       'movThreshold',
       'rpmThreshold',
```
Comment (trisapeace) [inline index.js:641]: Should you remove `rpmThreshold` as well?

File: index.js [hunk 3]
```diff
@@ -678,9 +675,6 @@ app.post(
           data.fallbackPhones.split(','),
           data.heartbeatPhones.split(','),
           data.twilioPhone,
-          data.sensitivity,
-          data.led,
-          data.noiseMap,
           data.movThreshold,
           data.rpmThreshold,
```
Comment (trisapeace) [inline index.js:679]: Should you remove `rpmThreshold` as well?

**Review by trisapeace — COMMENTED** [review]
Yay for code simplification (aka removal) ;-)

**Comment by mariocimet** [comment]
Made redundant by #113

**Pull request closed without merging**